Index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  <div class="container">
+    <nav class="navbar">
+      <ul class="nav-list header-menu">
+        <li ><a href="Index.html">Home</a></li>
+        <li ><a href="/About.html">About</a></li>
```

Comment:
In order for the about page to be properly linked on a remote server, kindly make the href a relative path instead of absolute. You could use `./About.html` instead of `/About.html`.